assignments/lambda-classes.js

```diff
@@ -1 +1,122 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(personStats){
+        this.name = personStats.name;
+        this.age = personStats.age;
+        this.location = personStats.location;
+        this.gender = personStats.gender;
+    }
+
+    speak() {
+        return `Hello, my name is ${this.name} and I am from ${this.location}.`;
+    }
+}
+
+class Instructor extends Person {
+    constructor(InstructorStats){
+        super(InstructorStats);
+        this.specialty = InstructorStats.specialty;
+        this.favLangauge = InstructorStats.favLangauge;
+        this.catchPhrase = InstructorStats.catchPhrase;
+    }
+
+    demo(subject) {
+        return `Today we are learning about ${subject}`;
+    };
+
+    grade(student, subject) {
```

Comment:
To clarify, yes, the student.name must be in the template string, but first the object `student` must be passed in... in the issue that Will had he was passing in the string and not the whole obj which lead to the error when he tried to return it in the block below